classes/dac-image-chocolate-doom.bbclass

```diff
@@ -0,0 +1,8 @@
+SUMMARY = "Base class for DAC chocolate-doom port"
```

Comment:
This class file is not necessary. Doom is standard SDL application, so it should inherit after dac-image-sdl.bbclass